components/feature/prompts/build.gradle

```diff
@@ -28,15 +28,23 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
 }
 
 dependencies {
+    // TODO this is only required for [DefaultLoginValidationDelegate]. Find a better home for that class and remove this dependency
+    implementation project(':service-sync-logins')
```

Comment:
(re this TODO) I haven't had any luck finding a home for `DefaultLoginValidationDelegate`.  Does anyone more familiar with ACs internal structure have any better ideas?